tracee-ebpf/Makefile

```diff
@@ -55,12 +61,17 @@ endif
 $(OUT_DIR):
 	mkdir -p $@
 
+.PHONY: $(OUT_BTFHUB)
+$(OUT_BTFHUB):
+	mkdir -p $@
+	touch $@/place-holder # can't embed empty dir
+
```

Comment:
it also creates "./dists/btfhub" directory and works the same way as the other target (for ./dist/libbpf, for example). The need for place-holder is because go does not support embedding empty dirs and without the file ./dist/btfhub directory would be empty. WDYT ?

Reply:
I think there is no need for both OUT_BTFHUB and btfhub targets, but maybe I'm missing something.
If possible, I would merge these two, and fix the problem with embedding empty dirs by using go build tags.
somehow related: #816

Reply:
Addressed your suggestion and pushed the new commit.